test/MUXControlsReleaseTest/NugetPackageTestApp/TestPages/NavigationViewTestPage.xaml

```diff
@@ -0,0 +1,14 @@
+﻿<utils:TestPage
```

Comment:
`<!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT License. See LICENSE in the project root for license information. -->`